src/testdrive/src/action/sql.rs

```diff
@@ -288,6 +288,17 @@ impl fmt::Display for ErrorMatcher {
     }
 }
 
+impl ErrorMatcher {
+    fn fmt_with_type(&self, type_: &str) -> String {
```

Comment:
Rust beginner question - why did you underscore-suffix `type_`? Because it's a string literal?